R/PlainTextParam.R

```diff
@@ -48,6 +51,11 @@
 #'   `readMsObject()` functions for other classes (such as `Spectra`,
 #'   `MsExperiment` etc).
 #'
+#' @param offline For `readMsObject()` to load MS data as a
+#'   `MsBackendMetaboLights()`: `logical(1)` to evaluate the local file cache
+#'   and only load local files. Thus `offline = TRUE` does not need an active
```

Comment:
I would avoid repetition maybe and instead maybe `..to evaluate **only** the local file cache`  ?

Reply:
Good point, thanks!

Reply:
Done with the latest commit.